### PR TITLE
Selfie: Implement camera mode switcher animation

### PR DESCRIPTION
**Pull request by luk1337**
Change-Id: If42b048f551c392bbac1991566f453287d5dc544

### CODE REVIEW

**Comment by luk1337** [comment]
https://user-images.githubusercontent.com/6830979/185717805-a5684d55-520d-429c-9248-4b5aba1c2169.mp4

**Comment by luk1337** [comment]
ehh, h265 doesn't work

**Comment by luk1337** [comment]
https://user-images.githubusercontent.com/6830979/185717887-813b7108-b06c-494d-a172-d53162a2fe12.mp4

File: app/src/main/java/org/lineageos/selfie/MainActivity.kt [hunk 1]
```diff
@@ -225,6 +227,15 @@ class MainActivity : AppCompatActivity() {
         micButton.setOnClickListener { toggleMicrophoneMode() }
         settingsButton.setOnClickListener { openSettings() }
 
+        // Initialize camera mode highlight position
+        qrModeButton.doOnLayout {
```
Comment (luk1337) [inline app/src/main/java/org/lineageos/selfie/MainActivity.kt]: would be nice if we didn't need to init anything ;(

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -282,6 +296,7 @@
             android:contentDescription="@string/photo_mode_button_description"
             android:padding="0dp"
             android:text="@string/selector_photo"
+            android:enabled="false"
```
Comment (luk1337) [inline app/src/main/res/layout/activity_main.xml]: did that just to make highlight look properly in android studio preview.

**Review by luca020400 — APPROVED** [review]

**Pull request merged**